BackEndApp/authv1/views.py

```diff
@@ -71,6 +71,9 @@ def register(request):
         message = "Some error occurred!! Please try again."
         status = 401
         token = None
+        return JsonResponse(
+            {"message": message, "username": None, "token": token}, status=status
+        )
```

Comment:
@mharyam Instead of returning the JsonResponse here, it would be better if you could just set username to `None` (following the way the other variables `status`, `token`... are updated)

Reply:
> @mharyam Instead of returning the JsonResponse here, it would be better if you could just set username to `None` (following the way the other variables `status`, `token`... are updated)

Hello, 
Thank you for the review

I have updated the pull request